src/services/NodeService/index.ts

```diff
@@ -0,0 +1,33 @@
+class NodeService {
```

Comment:
Maybe it would be better to call it `CKBNodeService`? Less misleading.

Reply:
Updated by https://github.com/Magickbase/nervos-official-website/pull/84/commits/f58d610c8680320b4e1dd85b62b80f17723e6b3b